addons/mail/static/src/messaging/component/composer_text_input/composer_text_input.xml

```diff
@@ -4,85 +4,21 @@
     <t t-name="mail.messaging.component.ComposerTextInput" owl="1">
         <div class="o_ComposerTextInput">
             <t t-if="composer">
-                <textarea class="o_ComposerTextInput_textarea" t-esc="composer.textInputContent" placeholder="Write something..." t-on-keydown="_onKeydownTextarea" t-on-input="_onInputTextarea" t-ref="textarea" />
+                <div class="dropup o_ComposerTextInput_mentionDropdown" t-att-class="{ 'o-lowPosition': props.mentionsLowPosition }">
+                    <div class="o_ComposerTextInput_mentionDropdownPropositionList dropdown-menu" t-att-class="{ 'o-lowPosition': props.hasMentionsLowPosition, 'show': composer.suggestedPartners.length > 0 }">
+                        <t t-foreach="composer.suggestedPartners" t-as="mention" t-key="mention.id">
+                            <PartnerMentionSuggestion
+                                isActive="_isActiveSuggestedPartner(mention)"
+                                partnerLocalId="mention.localId"
+                                t-on-o-partner-mention-suggestion-clicked="_onClickMentionItem"
+                                t-on-o-partner-mention-suggestion-mouse-over="_onMouseOverMention"
```

Comment:
cc @alexkuhn we could get rid of trigger up from `PartnerMentionSuggestion` if we allowed ourselves to define inline event handlers in template, we could just do something like `_onClickMentionItem(partner)` here. What do you think?